.storybook/decorators/index.js

```diff
@@ -93,6 +101,57 @@ export const withLanguageWrapper = makeDecorator({
 /**
  * @type import('@storybook/csf').DecoratorFunction<import('@storybook/web-components').WebComponentsFramework>
  **/
+export const withPlatformScaleWrapper = makeDecorator({
+	name: "withPlatformScaleWrapper",
+	parameterName: "context",
+	wrapper: (StoryFn, context) => {
+		const { args, globals, argTypes } = context;
+		const bodyEl = document.body;
+    const bodyElClasses = bodyEl.classList;
+		const mediumClass = 'spectrum--medium';
+		const largeClass = 'spectrum--large';
+		const SCALES = {
+			medium: 'medium',
+			large: 'large',
+			both: 'both',
+		};
+		const defaultScale = SCALES.medium;
+		const scale = globals.platformScale || argTypes.scale || args.scale || defaultScale;
+		const showBothScales = scale === 'both';
+
+		const updateBodyClasses = () => {
+			bodyElClasses.remove(mediumClass, largeClass);
+			bodyElClasses.add(`spectrum--${scale}`);
+		};
+
+		useEffect(() => {
+			if (scale) {
+				setTimeout(() => {
+					updateBodyClasses();
+        }, 100);
+			};
+		}, [scale]);
+
+		if (showBothScales) {
+			return html`
+				<div style="margin-bottom: 1rem" class=${mediumClass}>
```

Comment:
When displaying both scales on component where a popover is displayed, this space is not enough to also display the associated popover or causes some weirdness with how the components are displayed. 


Date Picker
![Screenshot 2023-12-18 at 6 13 33 PM](https://github.com/adobe/spectrum-css/assets/63808889/7cdb52c9-06db-4f21-879c-9615cb94c485)

Also, Alert Dialog, Combobox and Popover